### ISSUE TITLE
BABYSTEPPING not working?

### ISSUE THREAD

**Issue by edwilliams16** [MarlinFirmware/Marlin#4027]
I turned on BABYSTEPPING in Configuration_adv.h with

``` cpp
#define BABYSTEPPING
#if ENABLED(BABYSTEPPING)
  #define BABYSTEP_XY  //not only z, but also XY in the menu. more clutter, more functions
                       //not implemented for deltabots!
  #define BABYSTEP_INVERT_Z false  //true for inverse movements in Z
  #define BABYSTEP_MULTIPLICATOR 1 //faster movements
#endif
```

  The baby-stepping menus appeared on my LCD when printing, but stepping z up 0.4 mm with the arrow keys had no visible effect.  The print completed normally at exactly the same z height with no evidence of missing layers.

  I'm wondering, looking at the code, whether one click of the arrow keys is actually giving me one step in the z-direction where I have 1600 steps/mm and that I need to set the multiplier?  Will the distance displayed on the LCD screen be correct?

**Comment by edwilliams16** [comment]
OK.  It is a multiplier issue.  I set the multiplier to 16  and was able to lift z enough to have an effect.
However the LCD appears to display steps/10 - mm would be more useful, so we need a factor of 10/STEPS_PER_UNIT somewhere in the LCD display routine.

**Comment by edwilliams16** [comment]
I made a fix for this and submitted a pull request.  Since it's my first try at this git stuff be kind if I got the mechanics wrong.

**Comment by MrExo3D** [comment]
I would love to be able to set the control dial to by default control the babystep for Z. instead of the speed. when its in the idle menu. i never adjust speed. but i find i adjust babystepping often to get it started right.

can this be an option? 
let you choose the default behaviour of the control dial.

**Comment by thinkyhead** [comment]
@adamfilip That sounds like a separate Feature Request. Please post a new issue with that request.

**Comment by jbrazio** [comment]
Fixed by #4028
